Exoft.Gamification.Api.Data/Repositories/Repository.cs

```diff
@@ -24,24 +24,36 @@ public async Task<T> GetByIdAsync(Guid id)
             return await IncludeAll().SingleOrDefaultAsync(i => i.Id == id);
         }
 
-        public virtual async Task AddAsync(T entity)
+        public virtual async Task<T> AddAsync(T entity)
         {
```

Comment:
Well its not needed to return result of update for Update/Create for repository... For service - definitely, but not for repository